controller/src/main/java/org/jboss/as/controller/PathAddress.java

```diff
@@ -314,7 +314,11 @@ public PathAddress append(String key) {
      * @param create {@code true} to create the last part of the node if it does not exist
      * @return the submodel
      * @throws NoSuchElementException if the model contains no such element
+     *
+     * @deprecated manipulating a deep DMR node tree via PathAddress is no longer how the management layer works
```

Comment:
Should the deprecated message mention the correct alternative?

Reply:
@jmesnil I updated this with a pointer to Resource and OperationContext.[read|remove]Resource. TBH I don't think these have been used in many years as they haven't done anything useful since we introduced Resource for AS 7.0 or 7.1.